src/api/cristinProject.ts

```diff
@@ -0,0 +1,16 @@
+import Axios from 'axios';
```

Comment:
hva med å bruke `Api`-prefix for filer som har api-kall? slik at det blir `cristinProjectApi.ts`, på samme måte som vi har for actions og reducers